have-gethostbyname.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Jan Stary <hans@stare.cz>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#include <netdb.h>
+
+int
+main(void)
+{
+	struct hostent* host;
+	host = gethostbyname("columbia.edu");
```

Comment:
Is it a good idea to invoke a function that internally performs a DNS request? Would it not be possible to test for the presence of gethostbyname without invoking it and waiting for the DNS response to arrive (or timeout)?

Reply:
Yes, `localhost` is probably better. Thanks.